doc/modules/linear_model.rst

```diff
@@ -422,24 +421,24 @@ Least Angle Regression
 Least-angle regression (LARS) is a regression algorithm for
 high-dimensional data, developed by Bradley Efron, Trevor Hastie, Iain
 Johnstone and Robert Tibshirani. LARS is similar to forward stepwise
-regression. At each step, it finds the predictor most correlated with the
-response. When there are multiple predictors having equal correlation, instead
-of continuing along the same predictor, it proceeds in a direction equiangular
-between the predictors.
+regression. At each step, it finds the feature most correlated with the
```

Comment:
most correlated feature?

Reply:
I actually think "_the feature most correlated with the target_." is more clear.